web/src/pages/HomePage/HomePage.tsx

```diff
@@ -13,7 +13,7 @@ const HomePage = () => {
         </p>
         <div className="grid h-full max-h-[472px] w-full flex-1 grid-cols-1 gap-3">
           <Link
-            className="relative block h-full w-full rounded bg-[url('/images/ShareImage.webp')] bg-cover text-lg text-white shadow hover:shadow-white"
+            className="relative block h-full w-full rounded bg-[linear-gradient(to_bottom_right,rgba(0,0,0,0.51),rgba(0,0,0,.2)),url('/images/ShareImage.webp')] bg-cover text-lg text-white shadow hover:shadow-white"
```

Comment:
ok let's do something like this, so that it is completely transparent on the side opposite of the logo. 

`to_bottom_right,rgba(0,0,0,0.3),rgba(0,0,0,0)`